SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -626,6 +629,12 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 		return bSimulatedPlayersIsEnabled;
 	}
 
+	void SetAutoGenerateConfigEnabledState(bool IsEnabled);
```

Comment:
Can you add "EnabledForCloud" somewhere ? There is another flag to auto generate config somewhere, but for local deployments.